andela/Ah-backend-aquaman
# #164069238 Report an article

#### What does this PR do?
Adds a feature that enables authenticated users to report inappropriate articles
#### Description of Task to be completed?
add ReportedArticleModel
notify the author of reported abuse

#### How should this be manually tested?
* Clone the repo, cd into Ah-backend-aquaman and check out the branch ft-report-article-164069238
* pip install -r requirements.txt
* python manage.py migrate
* python manage.py runserver
* use postman to hit the endpoint `/articles/<slug>/report/`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#164069238](https://www.pivotaltracker.com/story/show/164069238)
#### Screenshots (if appropriate) 

`post a report`
![Screenshot 2019-03-18 at 20 01 29](https://user-images.githubusercontent.com/20795487/54549193-9f7c9f00-49ba-11e9-9669-52d5b0de6ef2.png)



`Email sent to the author`
![Screenshot 2019-03-18 at 20 05 28](https://user-images.githubusercontent.com/20795487/54549331-e2d70d80-49ba-11e9-9f07-f54b647cacc5.png)
